src/components/skybox.js

```diff
@@ -231,6 +231,14 @@ AFRAME.registerComponent("skybox", {
   init() {
     this.sky = new THREE.Sky();
     this.el.setObject3D("mesh", this.sky);
+    this.skyScene = new THREE.Scene();
+    this.cubeCamera = new THREE.CubeCamera(1, 100000, 512);
```

Comment:
Is 512 the best resolution to use? I see that the textures for our default environment map are 1024x1024. Perhaps we should inspect the environment map we have and decide at runtime?